vmware_dvs/utils/dvs_util.py

```diff
@@ -25,24 +25,33 @@
 
 from vmware_dvs.common import constants as dvs_const
 from vmware_dvs.common import exceptions
+from vmware_dvs.utils import spec_builder
 
 LOG = log.getLogger(__name__)
 
+CREATING_PG_STATUS = 'creating'
+READY_PG_STATUS = 'ready'
+UPDATING_PG_STATUS = 'updating'
+REMOVING_PG_STATUS = 'removing'
+
+INIT_PG_PORTS_COUNT = 4
+
+# TODO(ekosareva): get task_pool_interval from config
+TASK_POOL_INTERVAL = 2
+FREE_PORTS_CACHE_SIZE = 20
```

Comment:
I use FREE_PORTS_CACHE_SIZE to determine how many free_port keys stored in cache for each network. For example: network with 4096 keys, 2048 is free, but we will store in cache only 20.